src/System.Text.RegularExpressions/src/System/Text/ValueStringBuilder.Reverse.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text
+{
+    internal ref partial struct ValueStringBuilder
+    {
+        public void AppendReversed(ReadOnlySpan<char> value)
+        {
+            Span<char> span = AppendSpan(value.Length);
+            value.CopyTo(span);
+            span.Reverse();
+        }
+
+        public void Reverse(int start = 0, int length = 0)
```

Comment:
Do you actually use the start and length arguments anywhere? It is a pretty non-standard convention for length==0 to mean current length.

Reply:
No, it's just overhead. I removed the overloads.